quinn-proto/src/tests/util.rs

```diff
@@ -515,6 +520,16 @@ fn packet_size(transmit: &Transmit) -> usize {
     transmit.contents.len()
 }
 
+fn ecn(x: Option<EcnCodepoint>, congestion_experienced: bool) -> Option<EcnCodepoint> {
+    x.map(|codepoint| {
+        if congestion_experienced {
+            EcnCodepoint::from_bits(0b11).unwrap()
```

Comment:
`EcnCodepoint::Ce`? Those variants are only hidden for the benefit of the public API.